File: Debug/Barcode/Examples/VS2005/RDLC_Windows_CS/LineItem.cs
namespace RDLC_Windows_CS
{
    public class LineItem
    {
        private int _quantity;
        private string _product;
        private decimal _unitPrice;

        public string Product
        {
            get { return _product; }
            set { _product = value; }
        }

        public int Quantity
        {
            get { return _quantity; }
            set { _quantity = value; }
        }

        public decimal UnitPrice
        {
            get { return _unitPrice; }
            set { _unitPrice = value; }
        }
    };
}

File: Debug/Barcode/Examples/VS2010/RDLC_Windows_VB/LineItem.vb
Public Class LineItem
    Private m_product As String
    Private m_unitPrice As Decimal
    Private m_quantity As Integer

    Public Property Product() As String
        Get
            Return m_product
        End Get
        Set(ByVal value As String)
            m_product = value
        End Set
    End Property

    Public Property UnitPrice() As Decimal
        Get
            Return m_unitPrice
        End Get
        Set(ByVal value As Decimal)
            m_unitPrice = value
        End Set
    End Property

    Public Property Quantity() As Integer
        Get
            Return m_quantity
        End Get
        Set(ByVal value As Integer)
            m_quantity = value
        End Set
    End Property
End Class
